app/Commands/Dev/Core/Compile/Options.hs

```diff
@@ -1,6 +1,20 @@
 module Commands.Dev.Core.Compile.Options
-  ( module Commands.Extra.Compile.Options,
+  ( module Commands.Dev.Core.Compile.Options,
+    module Commands.Extra.Compile.Options,
   )
 where
 
 import Commands.Extra.Compile.Options
+import CommonOptions
+import Data.List.NonEmpty qualified as NonEmpty
+
+type CoreCompileOptions = CompileOptions
+
+coreSupportedTargets :: NonEmpty CompileTarget
+coreSupportedTargets = NonEmpty.fromList allTargets
```

Comment:
I think we should exclude `core` target here. It is just a silent no-op, which is confusing.

Reply:
Agree, I'll make the list explicit here.

Reply:
Done.